pkg/koordlet/prediction/peak_predictor.go

```diff
@@ -142,10 +151,10 @@ func (p *podReclaimablePredictor) AddPod(pod *v1.Pod) error {
 	}
 
 	if p.pods[string(pod.UID)] {
-		return fmt.Errorf("pod %v already exists", pod.UID)
-	} else {
-		p.pods[string(pod.UID)] = true
+		klog.V(6).Infof("Pod %s already exist in the pod reclaimable predictor", util.GetPodKey(pod))
+		return nil
```

Comment:
It is not recommended to eat this error.

Reply:
@hormes If this error message is not removed, will the amount of error messages be too much?

Reply:
@hormes return error code rolled back.